File: python/ejer35.py
#solicitar numero y calcular su factorial
numero = int(input("Ingrese un número para calcular su factorial: "))
factorial = 1

if numero < 0:
    print("El factorial no está definido para números negativos.")
elif numero == 0 or numero == 1:
    print(f"El factorial de {numero} es 1.")
else:
    for i in range(2, numero + 1):
        factorial *= i
    print(f"El factorial de {numero} es: {factorial}")

File: js/ejer35.js
//solicitar numero y calcular su factorial
let numero = parseInt(prompt("Ingrese un numero para calcular su factorial:"));
let factorial = 1;  
if (numero < 0) {
    document.write("El factorial no esta definido para numeros negativos.");
} else if (numero === 0 || numero === 1) {
    document.write("El factorial de " + numero + " es 1.");
} else {
    for (let i = 2; i <= numero; i++) {
        factorial *= i;
    }
    document.write("El factorial de " + numero + " es: " + factorial);
}   
